templates/topic.tpl

```diff
@@ -34,7 +34,7 @@
 		<ul component="topic" class="posts" data-tid="{tid}" data-cid="{cid}">
 			<!-- BEGIN posts -->
 				<li component="post" class="<!-- IF posts.deleted -->deleted<!-- ENDIF posts.deleted -->" <!-- IMPORT partials/data/topic.tpl -->>
-					<a component="post/anchor" data-index="{posts.index}" name="{posts.index}"></a>
+					<a component="post/anchor" data-index="{posts.index}" id="{posts.index}"></a>
```

Comment:
I think `id` is necessary for inter-page traversal via anchors?

Reply:
I'm confused...
I replaced name with id, if you mean it the other way round then I just can say, that name is a "legacy reserved" keyword for all elements, and I haven't found any reference to this attribute in the nodebb core code.

Reply:
Both `id` and `name` work with anchor traversal, @julianlam